Sage/carbon
# fix(pager): ensure id passed to page size select is unique

fix #4947

### Proposed behaviour


Adds guid to `id` passed to page size `Select`

### Current behaviour


`id` for page size `Select` is not unique

### Checklist




#### QA


### Additional context



### Testing instructions



The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.


https://codesandbox.io/s/pager-size-options-mwpmf3?file=/src/index.js -- test the build version of this
